apps/designer/src/env/client.mjs

```diff
@@ -0,0 +1,33 @@
+// @ts-nocheck
```

Comment:
why are these files needed?

Reply:
reference :https://github.com/t3-oss/create-t3-turbo/

other prisma and trpc won't work with the next.js and dynamic module import will not work